ValheimPlus/Game.cs

```diff
@@ -60,5 +60,48 @@ private static Boolean Prefix(ref Game __instance, ref Vector3 pos, ref int __re
             return true;
         }
     }
+    
+    [HarmonyPatch(typeof(Game), "EverybodyIsTryingToSleep")]
+    public static class Game_EverybodyIsTryingToSleep
+    {
+        private static bool Postfix(bool _result, ref ZDO __instance)
+        {
+            if (Configuration.Current.Sleep.IsEnabled)
+            {
+                if (Configuration.Current.Server.IsEnabled)
```

Comment:
Why are we checking that Server is enabled here?

Reply:
Good point, removing the check.